### ISSUE TITLE
Not an issue, just a suggestion on a cool and dirty little hack I've been working on for the past weeks!

### ISSUE THREAD

**Issue by Deus-nsf** [fholger/farcry_vrmod#35]
Player Shadows!
https://youtu.be/7k3CCra-1qc?si=keqFMEIEIiZdHZsJ

here are the files I modified in the SDK to make it possible, don't hesitate to diff those!
https://mega.nz/file/qh5zxLqY#-EH6jcM0A9QSwG4YievKXuI8FL_3xn86HC0ce6WpqVU

Other cool things to salvage if you're up for a little digging action (like better reflections from the water shader, etc...): 
https://mega.nz/file/6lwGyDiZ#OE8akt5L1bNi-HvVz4a0h8_Tbp6vLLyOIrPNQdXNMcA

**Comment by NoUserNameForYou** [comment]
If devs don't add it, is there any possiblity you could?

**Comment by Deus-nsf** [comment]
I don't think so, but I trust the maintainer will like it and find a way wink wink especially since there's not a lot of code involved wink wink :D

**Comment by NoUserNameForYou** [comment]
Rip :( 1.0.0 is feature set complete.

**Comment by Deus-nsf** [comment]
Rest in pepperoni! No shadows in VR! To be fair, I've always been very torn with player shadow maps in outdoors, however player stencil shadows indoor look just incredible, it's a shame, but maybe it doesn't play well in VR? Who knows.
Or maybe the maintainer just didn't see this thread lol.
